common/base/SharedMemory_posix.cpp

```diff
@@ -19,9 +19,7 @@
 #include "gfxstream/EintrWrapper.h"
 #include "gfxstream/memory/SharedMemory.h"
 #include "gfxstream/files/PathUtils.h"
```

Comment:
You don't need to build `SharedMemory_posix.cpp` at all for Windows

if host_machine.system() == "windows"
     files_lib_common_base += files ('SharedMemory_win32.cpp`)
else
     files_lib_common_base += files ('SharedMemory_posix.cpp`)
endif